app/config/set_parameters.php

```diff
@@ -59,5 +59,8 @@
     }
 
     $container->setParameter('cache.driver', extension_loaded('apc') ? 'apc': 'array');
+
+    // Parameter used only in dev and test env
+    $container->setParameter('use_debug_toolbar', !getenv('DISABLE_DEBUG_TOOLBAR'));
```

Comment:
This line erased what you defined in parameters.yml
```suggestion
    if (getenv('DISABLE_DEBUG_TOOLBAR') !== false) {
        $container->setParameter('use_debug_toolbar', !getenv('DISABLE_DEBUG_TOOLBAR'));
    }
```

Reply:
@PierreRambaud done! Let's wait for the build to see if it works correctly